builds-gcc/Dockerfile.centos7-gcc9

```
@@ -0,0 +1,35 @@
+ARG FROM_IMAGE=nvidia/cuda
+ARG CUDA_VER=11.0
+ARG CUDA_TYPE=devel
+ARG LINUX_VER=centos7
+FROM ${FROM_IMAGE}:${CUDA_VER}-${CUDA_TYPE}-${LINUX_VER}
+
+# Define arguments
+ARG GCC9_DIR=/usr/local/gcc9
+ARG GCC9_VER=9.3.0
+ARG NUM_BUILD_CPUS=16
+
+# Add /usr/local/cuda/* temporarily to LD_LIBRARY_PATH to support various build steps
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH_POSTBUILD:/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs
+
+RUN yum upgrade -y \
+    && yum install -y \
```

```suggestion
    && yum install -y --setopt=install_weak_deps=False \
```

Can you add this change to the new Dockerfile? This prevents some extraneous dependencies from being installed that triggered our security scans. See #174 for more details.

---

Done